### ISSUE TITLE
Localização pt-BR

### ISSUE THREAD

**Issue by andrebaltieri** [lira92/flunt.br#36]
Na v2 do Flunt teremos mensagens padrão nas notificações:
https://github.com/andrebaltieri/Flunt/blob/dev/Flunt/Localization/ContractLocalization.cs

O que acha de incluirmos as mensagens pt-BR aqui neste pacote?

**Comment by lira92** [comment]
@andrebaltieri certo, quando fizer a adequação para a v2, coloco sim. Mas temos que alinhar em como mudar o locale, pelo que vi atualmente não tem uma forma de mudar isso né?

**Comment by andrebaltieri** [comment]
https://balta.io/blog/flunt => veja na sessão Localização
